lib/src/replay/mask/posthog_nomask_widget.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class PostHogNoMaskWidget extends StatefulWidget {
```

Comment:
```suggestion
class PostHogMaskWidget extends StatefulWidget {
```

Its about masking and not `not` Masking, I think its a typo in the issue description, sorry.